README.md

```diff
@@ -340,6 +341,28 @@ func isError(error: Error?) -> Bool {
 }
 ```
 
+### Implicit Getters ###
+
+When overriding only the getter of a property, omit the use of `get`:
```

Comment:
Yeah, or you can see it as gentle reminder for people who don't think in terms of compiler or would be wondering and don't want to have to open xcode to find out